crates/wysiwyg/src/dom/range.rs

```diff
@@ -51,6 +52,9 @@ pub struct DomLocation {
     /// True if this is a node which is not a container i.e. a text node or
     /// a text-like node like a line break.
     pub is_leaf: bool,
```

Comment:
I'm guessing this can be computed from `DomNodeKind` now ?

Reply:
Yes, nice catch!